bounded-collections/src/bounded_btree_map.rs

```diff
@@ -123,6 +123,14 @@ where
 	}
 }
 
+impl<K, V, S> DecodeWithMemTracking for BoundedBTreeMap<K, V, S>
```

Comment:
I thought that `BoundedBtreeMap` is just a wrapper over `BTreeMap`, but I see it has its own implementation for decode. So we should do the same as for in the decoding logic: `BtreeMap`: https://github.com/paritytech/parity-scale-codec/blob/8d65d28e778d914988849b786c17216070ecd9a1/src/codec.rs#L1273